### ISSUE TITLE
Padding and trimming sentences

### ISSUE THREAD

**Issue by versusnja** [keras-team/keras#7901]
I would suggets that instead of 
`data = pad_sequences(sequences, maxlen=MAX_SEQUENCE_LENGTH)`

it must be
`data = pad_sequences(sequences, maxlen=MAX_SEQUENCE_LENGTH, truncating='post', padding='post')`

Assuming that words in the beginning of a sentence contain "more signal" / "more meaning" than the words in the end. 

https://github.com/fchollet/keras/blob/8c8db2cd9ac263e082f8804899e97b7b7e910bda/examples/pretrained_word_embeddings.py#L87
